src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalTool.java

```diff
@@ -174,7 +174,9 @@ public JsonObjectBuilder toJson() {
         // from https://swagger.io/specification/#fixed-fields-29 but that's for URLs.
         FILE_ID("fileId"),
         SITE_URL("siteUrl"),
-        API_TOKEN("apiToken");
+        API_TOKEN("apiToken"),
+        DATASET_ID("datasetId"),
+        DATASET_VERSION("datasetVersion");
```

Comment:
@qqmyers can you please add these two new reserved words (datasetId and datasetVersion) to the list of possibilities? The current list can be found at http://guides.dataverse.org/en/4.10.1/installation/external-tools.html